README.md

```diff
@@ -16,6 +16,7 @@ import { carbonFootprint } from 'trip-to-carbon-xyz'
 
 const taxiFootprint = await carbonFootprint({
   country: 'USA',
+  token: 'YourAppToken',  //remove this property if you do not have a token
```

Comment:
Would you mind:

1. moving this to the top (as the first option)
1. changing the comment to just "optional"